.drone.yml

```diff
@@ -354,46 +354,72 @@ steps:
     when:
       cron: security_scans
       event: cron
-
-  - name: cron_trivy_scan
+  
+  - name: cron_trivy_scan_image_os
     image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
     pull: always
     environment:
-      IMAGE_NAME: hff:${DRONE_COMMIT_SHA}
-      SEVERITY: MEDIUM,HIGH,CRITICAL
-      FAIL_ON_DETECTION: false
-      IGNORE_UNFIXED: true
-      ALLOW_CVE_LIST_FILE: hof-services-config/Employers_Checking_Service/trivy-cve-exceptions.txt
+        IMAGE_NAME: node:20.15.0-alpine3.20@sha256:24c14a8a192a6e81d0942929a344f7a4bdf0db8e3b3c77d64a5eb8a4b0c759b7
```

Comment:
need to update the node version to match the one in the Docker file

Reply:
Well spotted, thank you